testgrid/istio-generated.yaml

```diff
@@ -29,23 +29,715 @@ default_test_group:
   use_kubernetes_client: true
```

Comment:
A new version of Configurator should be able to regenerate without the default settings; see https://github.com/kubernetes/test-infra/pull/13636

You can include the default settings, but they won't be respected; the defaults are set by instance of TestGrid